game/src/main/java/org/apollo/game/model/entity/obj/GameObject.java

```diff
@@ -85,6 +86,23 @@ public int getType() {
 		return packed >> 2 & 0x3F;
 	}
 
+
+	@Override
+	public int getHeight() {
+		Direction direction = Direction.WNES[getOrientation()];
+
+		return direction == Direction.WEST || direction == Direction.EAST ?
```

Comment:
This is used in a lot of places... (traversal stuff) maybe we could extract this functionality into its own method

Reply:
I think if we started using these new functions in the traversal stuff we wouldn't need another method, right? Assuming that code is getting the width / height of the object.

Reply:
Correct